jobQueue/setup.js

```diff
@@ -45,6 +45,7 @@ module.exports = {
     JOB_CLEANUP: 1,
     PERSIST_SAMPLE_STORE: 1,
     SAMPLE_TIMEOUT: 1,
+    BULKCREATE_AUDITEVENTS: 1,
```

Comment:
name should be `BULK_CREATE_AUDIT_EVENTS`

Also: we want to be able to process more than one of these at a time (imagine there are dozens of collectors sending POST auditevents every few seconds!), so we should set up a concurrency env var for it